### PR TITLE
[l10n] Add Ukrainian (uk-UA) locale on the date picker

### PR DESCRIPTION
**Pull request by Dufran**

### CODE REVIEW

File: packages/x-date-pickers/src/locales/ukUA.ts [hunk 1]
```diff
@@ -0,0 +1,75 @@
+import { PickersLocaleText } from './utils/pickersLocaleTextApi';
+import { getPickersLocalization } from './utils/getPickersLocalization';
+import { CalendarPickerView } from '../internals/models';
+
+// This object is not Partial<PickersLocaleText> because it is the default values
+
+const ukUAPickers: PickersLocaleText<any> = {
```
Comment (alexfauquette) [inline packages/x-date-pickers/src/locales/ukUA.ts:7]:
```suggestion
const ukUAPickers: Partial<PickersLocaleText<any>> = {
```

**Review by alexfauquette — COMMENTED** [review]
To le make it work, it need to be export from `locale/index.ts`.

And you can add a line to [the table of supported locales](https://github.com/mui/mui-x/blob/master/docs/data/date-pickers/localization/localization.md) to say Ukrainian is supported

**Review by alexfauquette — APPROVED** [review]
Thanks for your contribution, every think sounds nice :)

The CI seems flaky, I will add an empty commit, to see if let it run

**Comment by Dufran** [comment]
@alexfauquette 
Thanks for help through the process. It still seems to be some errors in `ci/circleci: test_static`  job, did I miss something?

**Comment by alexfauquette** [comment]
You can run `yarn docs:api` and commit changes. It will add `ukUA` to the list of exported  variables

**Comment by Dufran** [comment]
@alexfauquette Added changes can you please kick CircleCI Pipeline one more time

File: packages/x-date-pickers/src/locales/ukUA.ts [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+  timePickerToolbarTitle: 'Вибрати час',
+  dateRangePickerToolbarTitle: 'Вибрати календарний період',
+
+  // Clock labels
+  clockLabelText: (view, time, adapter) =>
+    `Select ${view}. ${
+      time === null ? 'Час не вибраний' : `Вибрано час ${adapter.format(time, 'fullTime')}`
+    }`,
+  hoursClockNumberText: (hours) => `${hours} годин`,
+  minutesClockNumberText: (minutes) => `${minutes} хвилин`,
```
Comment (cherniavskii) [inline packages/x-date-pickers/src/locales/ukUA.ts:46]: As potential improvement, we can use different plural forms depending on the number like we do in the data grid:
https://github.com/mui/mui-x/blob/099729f6704570575b0a395a6efac471449d0888/packages/grid/x-data-grid/src/locales/ukUA.ts#L46-L51

Comment (alexfauquette) [reply packages/x-date-pickers/src/locales/ukUA.ts:46]: Sorry, I saw your profile picture, then I thought you did a language review but it was your self-assignment 🙈

Comment (cherniavskii) [reply packages/x-date-pickers/src/locales/ukUA.ts:46]: @alexfauquette no worries 🙂
@Dufran Feel free to open a follow-up PR if you want

**Pull request merged**